go.mod

```diff
@@ -149,3 +151,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20211208200746-9f7c6b3444d2 // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.1 // indirect
 )
+
+replace github.com/ahmetb/gen-crd-api-reference-docs => github.com/tektoncd/ahmetb-gen-crd-api-reference-docs v0.3.1-0.20220729140133-6ce2d5aafcb4 // Waiting for https://github.com/ahmetb/gen-crd-api-reference-docs/pull/43/files to merge
```

Comment:
Is the comment here still relevant? do you need the `replace`?

Reply:
Yes, because the package in the forked repo is still `github.com/ahmetb/gen-crd-api-reference-docs`.